Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -633,11 +633,11 @@
     <!-- XACT: settings(background priority) - illustraction description -->
     <string name="settings_background_priority_illustration_description"/>
     <!-- XTXT: settings(background priority) - explains user what to do on card if background priority is enabled -->
-    <string name="settings_background_priority_card_body">"To disable prioritized background activity, please deactivate it in the device settings."</string>
+    <string name="settings_background_priority_card_body">"To enable / disable prioritized background activity, please change it in the device settings"</string>
```

Comment:
We don't change the english strings, that has to go over the translation process.

Reply:
please revert all changes in this file.